packages/trace-viewer/src/ui/callTab.css

```diff
@@ -105,3 +105,7 @@ a.call-value:hover {
   padding: 5px;
   line-height: 17px;
 }
+
+.metadata-view {
```

Comment:
Let's inline the style and remove this class. Migrate to data-test='metadata-view' for tests